app/build.gradle

```diff
@@ -62,8 +65,20 @@ android {
             }
 
             manifestPlaceholders = [appName: "Vision Teaser"]
+
+            firebaseCrashlytics {
+                nativeSymbolUploadEnabled true
+                strippedNativeLibsDir 'build/intermediates/stripped_native_libs/release/out/lib/'
+                unstrippedNativeLibsDir '../source/mapbox-vision/SDK/Platforms/Android/build/unstripped/release'
+            }
         }
     }
+
+    dexOptions {
```

Comment:
Do we need those dexOptions? Isn't xmx=8g in properties enough?

Reply:
Properties are enough, but why not speed up dex-ing?

Reply:
Indeed